diff -Nrc mooreutils/envx.c moreutils/envx.c
*** mooreutils/envx.c	2015-12-24 17:54:01.689571301 +0400
--- moreutils/envx.c	1970-01-01 03:00:00.000000000 +0300
***************
*** 1,27 ****
- #include <stdio.h>
- # include <stdlib.h>
- # include <string.h>
- int main(int argc, char const *argv[])
- {
- 
- 	char str[80];
- 	char* pid = argv[1];
- 
- 	if( argc<2 ) return 1;
- 
- 	strcpy(str, "/proc/");
- 	strcat(str, pid);
- 	strcat(str, "/environ");
- 
- 	char ch;
- 	FILE *fp;
- 	fp = fopen(str,"r");
- 	printf("Text from file:%s\n", str);
- 
- 	while( ( ch = fgetc(fp) ) != EOF )
- 	printf("%c",ch);
- 
- 	fclose(fp);
- 	printf("\n");
- 	return 0;
- }
--- 0 ----
Binary files mooreutils/.git/index and moreutils/.git/index differ
Binary files mooreutils/.git/objects/pack/pack-fa3d0c18181a14becf556bb5ca1f61eb5227a483.idx and moreutils/.git/objects/pack/pack-fa3d0c18181a14becf556bb5ca1f61eb5227a483.idx differ
Binary files mooreutils/.git/objects/pack/pack-fa3d0c18181a14becf556bb5ca1f61eb5227a483.pack and moreutils/.git/objects/pack/pack-fa3d0c18181a14becf556bb5ca1f61eb5227a483.pack differ
